### DISCUSSION TITLE
Is this project still being worked on?

### DISCUSSION THREAD

**Discussion by Daza99** [luiscastilho/dungeoneering#10] [General]
Came across this today and looks what I have been hoping to find. Nice and simple interface.

Just curious to know if it is still being worked on or considered finished?

**Comment by luiscastilho** [comment]
Hi there. Glad to hear you liked it!
It's a one man project and it's not a priority for me right now so development is quite slow. But yeah, I intend to keep working on it, specially to fix bugs and make it more intuitive and easier to use.
